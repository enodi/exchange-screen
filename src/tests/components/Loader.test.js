import React from "react";
import { shallow } from "enzyme";
import Loader from "components/Loader";

describe("Loader Component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Loader />);
  });

  it("should render Loader component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
