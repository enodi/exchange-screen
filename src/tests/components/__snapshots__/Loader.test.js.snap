// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component should render Loader component correctly 1`] = `
<span
  className="loading__anim"
/>
`;
